Drop after lang/rust is updated to 1.53 or later
https://github.com/rust-lang/rust/pull/79278

error[E0658]: or-patterns syntax is experimental
   --> common/src/comp/inventory/item/mod.rs:858:14
    |
858 |               (SpriteKind::DungeonChest0
    |  ______________^
859 | |             | SpriteKind::DungeonChest1
860 | |             | SpriteKind::DungeonChest2
861 | |             | SpriteKind::DungeonChest3
...   |
866 | |             | SpriteKind::Crate
867 | |             | SpriteKind::ChestBuried) => {
    | |_____________________________________^
    |
    = note: see issue #54883 <https://github.com/rust-lang/rust/issues/54883> for more information
    = help: add `#![feature(or_patterns)]` to the crate attributes to enable

error[E0658]: or-patterns syntax is experimental
  --> voxygen/anim/src/biped_large/spin.rs:93:21
   |
93 |         if let Some(ToolKind::Axe | ToolKind::Hammer | ToolKind::Dagger) = active_tool_kind {
   |                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: see issue #54883 <https://github.com/rust-lang/rust/issues/54883> for more information
   = help: add `#![feature(or_patterns)]` to the crate attributes to enable

error[E0658]: or-patterns syntax is experimental
   --> voxygen/src/audio/sfx/mod.rs:337:25
    |
337 | /                         object::Body::Arrow
338 | |                         | object::Body::MultiArrow
339 | |                         | object::Body::ArrowSnake
340 | |                         | object::Body::ArrowTurret,
    | |___________________________________________________^
    |
    = note: see issue #54883 <https://github.com/rust-lang/rust/issues/54883> for more information
    = help: add `#![feature(or_patterns)]` to the crate attributes to enable

error[E0658]: or-patterns syntax is experimental
   --> world/src/sim/map.rs:237:22
    |
237 |         (None | Some(RiverKind::Lake { .. } | RiverKind::Ocean), _) => Rgb::new(
    |                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |
    = note: see issue #54883 <https://github.com/rust-lang/rust/issues/54883> for more information
    = help: add `#![feature(or_patterns)]` to the crate attributes to enable

--- common/src/lib.rs.orig	2021-06-12 08:55:34 UTC
+++ common/src/lib.rs
@@ -12,6 +12,7 @@
     iter_map_while,
     label_break_value,
     option_zip,
+    or_patterns,
     trait_alias,
     type_alias_impl_trait
 )]
--- voxygen/anim/src/lib.rs.orig	2021-06-12 08:55:34 UTC
+++ voxygen/anim/src/lib.rs
@@ -1,5 +1,6 @@
 #![feature(const_generics)]
 #![feature(generic_associated_types)]
+#![feature(or_patterns)]
 #![allow(incomplete_features)]
 #[cfg(all(feature = "be-dyn-lib", feature = "use-dyn-lib"))]
 compile_error!("Can't use both \"be-dyn-lib\" and \"use-dyn-lib\" features at once");
--- voxygen/src/lib.rs.orig	2021-06-12 08:55:34 UTC
+++ voxygen/src/lib.rs
@@ -8,6 +8,7 @@
     const_generics,
     drain_filter,
     once_cell,
+    or_patterns,
     trait_alias
 )]
 #![recursion_limit = "2048"]
--- world/src/lib.rs.orig	2021-06-12 08:55:34 UTC
+++ world/src/lib.rs
@@ -13,6 +13,7 @@
     const_generics,
     const_panic,
     label_break_value,
+    or_patterns,
     array_map
 )]
 
